File: app/src/main/java/com/hundaol/ethiopic/views/HeaderItemViewHolder.java
package com.hundaol.ethiopic.views;

import android.support.v7.widget.RecyclerView;

import com.hundaol.ethiopic.viewmodels.HeaderItemViewModel;

/**
 * Created by abinet on 6/6/17.
 */

public class HeaderItemViewHolder extends RecyclerView.ViewHolder {

    private HeaderItemView view;

    public HeaderItemViewHolder(HeaderItemView view) {
        super(view);
        this.view = view;
    }

    public void setViewModel(HeaderItemViewModel viewModel) {
        view.bindTo(viewModel);
    }
}

File: app/src/main/java/com/hundaol/ethiopic/views/HeaderItemViewHolder.kt
package com.hundaol.ethiopic.views

import android.support.v7.widget.RecyclerView

import com.hundaol.ethiopic.viewmodels.HeaderItemViewModel

/**
 * Created by abinet on 6/6/17.
 */

class HeaderItemViewHolder(private val view: HeaderItemView) : RecyclerView.ViewHolder(view) {

    fun setViewModel(viewModel: HeaderItemViewModel) {
        view.viewModel = viewModel
    }
}
